modules/tinymce/Gruntfile.js

```diff
@@ -175,6 +179,37 @@ module.exports = function (grunt) {
             }
           ]
         };
+      }),
+      gruntUtils.generate(models, 'model', (name) => {
+        return {
+          options: {
+            treeshake: true,
+            format: 'iife',
+            onwarn: swag.onwarn,
+            plugins: [
```

Comment:
```suggestion
            plugins: [
              FilesAsStrings,
```
I assume we're going to need this here as well given it was added elsewhere?

Reply:
I doubt it's needed, really only the theme makes use of it, but yes it should go here.